arch/risc-v/src/esp32c3/esp32c3_start.c

```diff
@@ -83,7 +84,15 @@ void __esp32c3_start(void)
 
   showprogress('B');
 
-  /* Call nx_start() */
+  /* Set CPU frequency */
+
+  esp32c3_clockconfig();
```

Comment:
Shouldn't the clock configuration be performed before the call to `esp32c3_lowsetup()`?